### ISSUE TITLE
False-positive WRONG_WHITESPACE on nested braces inside lambda

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#283]
1. 
    ```kotlin
    list.forEach {
        try {
            foo()
        }
    }
    ```
    causes
    ```
    [WRONG_WHITESPACE] incorrect usage of whitespaces for code separation: there should be no whitespace before '{' of lambda inside argument list
    ```
    on a brace after `try`

2. Lambda passed to a named argument
    ```kotlin
    Example(cb = { _, _ -> Unit })
    ```

3. Lamda as a second argument
    ```kotlin
        foo(bar, { it.baz() })
    ```
